lms/static/js/learner_dashboard/views/program_details_view.js

```diff
@@ -7,16 +7,30 @@
             'gettext',
             'edx-ui-toolkit/js/utils/html-utils',
             'js/learner_dashboard/views/program_header_view',
+            'js/learner_dashboard/views/collection_list_view',
+            'js/learner_dashboard/views/course_card_view',
             'text!../../../templates/learner_dashboard/program_details_view.underscore'
            ],
-         function(Backbone, $, _, gettext, HtmlUtils, HeaderView, pageTpl) {
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             HtmlUtils,
+             HeaderView,
+             CollectionListView,
+             CourseCardView,
+             pageTpl
+         ) {
             return Backbone.View.extend({
                 el: '.js-program-details-wrapper',
 
                 tpl: HtmlUtils.template(pageTpl),
 
                 initialize: function(options) {
                     this.programModel = new Backbone.Model(options);
+                    this.context = options;
+                    this.courseCardsCollection = new Backbone.Collection(this.context.course_codes);
```

Comment:
@AlasdairSwan `this.model` is undefined in the view because you assigned the model to `this.programModel`. Was your intention to assign the model to `this.model` instead?